### DISCUSSION TITLE
Terminal - unable to key in password

### DISCUSSION THREAD

**Discussion by halley276** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#63] [Q&A] [answered]
Hi everyone. I am having a slight issue with my terminal.

Under the code management part of the GitHub wiki, we were asked to link Git to our Azure environment. Although I followed all the instructions, I had an issue when it came to entering my Github password for authentication. At that point, I was not able to key in my password as it seems to have hanged and was not recording what I was typing. Only when I use the “enter” key, it leads to “Authentication failed, invalid username or password”. 

For context, I am using a Chrome browser on a MacOS computer. I have also attached a screenshot of the error below. Thank you in advance!

<img width="774" alt="Screenshot 2021-05-19 at 9 45 47 PM" src="https://user-images.githubusercontent.com/65350949/118902517-66e50100-b948-11eb-9dc4-8d064ba4019e.png">

**Comment by EY-glowglowglow** [comment]
@codeindulgence can you please have a look at this?

**Comment by nickbutler** [comment, marked as answer]
Hi @halley276, it is common in command line environments for password entry to not show any input being typed.
When the password prompt shows, just type your password like you would normally, then hit return.

Thanks and let us know how you get on!

**Reply by halley276** [reply]
Thanks, I didn't know that!
